RPIPs/RPIP-43.md

```diff
@@ -1,7 +1,7 @@
 ---
 rpip: 43
 title: Megapools
-description: A single contract per node operator that can be used as an ethereum withdrawal address for multiple validators
+description: A single contract per node operator that can be used as an ethereum withdrawal address for multiple validators. Also allows for ETH-only validators.
```

Comment:
Also allows for ETH-only validators. -> Also allows for ETH-only node operation.

RPL has had an indirect tie to the validators anyhow -- being staked to the node or the megapool, but not linked to specific validators. I think this works better

Reply:
The other choice that would work would be making it about validator deposits or otherwise creating new validators